### ISSUE TITLE
Troubleshoot Option

### ISSUE THREAD

**Issue by lloyd094** [hang-up/buuk#18]
When compiling (especially when using plugins), there are times the wiki is incorrectly rendered, resulting in no errors, but a blank page (even after clearing cache).

Is there a way to show errors to better pinpoint what's happening or why?

**Comment by hang-up** [comment]
Are there any errors in the console? It'd be cool if you could send a screenshot of the console output!
I can't think of why it would render a white page..

**Comment by lloyd094** [comment]
Never thought to look at the console. Not much help to me, but perhaps you'll be able to understand it better?

I'm using the latest stable FireFox on Ubuntu.

![screenshot from 2017-06-29 16 17 22](https://user-images.githubusercontent.com/22482891/27714669-439393a4-5ce7-11e7-8ab1-bd7dba857963.png)

**Comment by hang-up** [comment]
I dug a bit and think I found the issue; nothing wrong with what you did but rather with the plugin itself (assuming it's `markdown-it-github-toc`)

The author of the plugin, is exporting his plugin using ES6 `export default` feature. At the moment, no environment natively supports this feature (https://caniuse.com/#search=export). And since we are inside the `node_modules` folder, Babel can't convert the module into a CommonJS module. 

I'll submit a pull request to the author of this plugin (as well as for `markdown-it-mermaid`, both are from the same author). In the meantime, there is not much we can do unfortunately...

**Comment by lloyd094** [comment]
That makes a ton of sense. Glad I didn't mess up on something. Thanks again!

**Comment by hang-up** [comment]
Implementing `markdown-it-toc` in Bük has been revealed to be more of a challenge than expected (conflicts with `vue-router`). This feature might eventually come up in the next minor release.
The good news is I talked to the developper of `markdown-it-mermaid` and `markdown-it-latex`, and we were able to come up with a solution that will hopefully fix the webpack related issue you faced earlier this week! 

I'll mark this issue as Closed therefore.
